pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -337,3 +375,52 @@ func gitRefInfo(repo git.Repository, dir, ref string) (buildapi.GitRefInfo, erro
 	info.Message = lines[4]
 	return info, nil
 }
+
+// WaitForBuildComplete waits for a build identified by the name to complete
+func WaitForBuildComplete(c osclient.BuildInterface, name string) error {
```

Comment:
I see duplicate code with the buildlog registry

Can't you factor out common parts to util functions? Follow-up is fine